### DISCUSSION TITLE
HTTPS client authentication

### DISCUSSION THREAD

**Discussion by axlgit** [bauerj/paperless_app#43] [Q&A]
Hi,

I haven't tried it yet (I haven't configured a proxy I use VPN for the moment) but does the PL-ng app support client authentication with certificate?

Alex

**Comment by bauerj** [comment]
Hey,

probably not (I haven't tried it either). Since custom server certificates don't work either I would be surprised if this worked. This seems to be a limitation of flutter at the moment.

Feel free to test it and let us know whether it works or not.

**Reply by axlgit** [reply]
Ok, I'll put in my todo list :-)
